pkg/kube/client.go

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
All the contents of `pkg/kube/*` extracted from `istio/pkg/kube/*`? If so, I wonder if it is worthwhile to maintain so much code by ourselves just to reduce the binary size.

Reply:
> All the contents of `pkg/kube/*` extracted from `istio/pkg/kube/*`? If so, I wonder if it is worthwhile to maintain so much code by ourselves just to reduce the binary size.

No, not all of it, just part of the code, though it does include quite a bit. The main goal was to provide only the methods needed by ctl.